modules/swagger-codegen/src/main/java/io/swagger/codegen/examples/ExampleGenerator.java

```diff
@@ -179,6 +179,7 @@ private Object resolvePropertyToExample(String propertyName, String mediaType, P
             if (innerType != null) {
                 int arrayLength = null == ((ArrayProperty) property).getMaxItems() ? 2 : ((ArrayProperty) property).getMaxItems();
                 if(arrayLength>10000) {
+                    logger.warn("The max item is too large to new Object array and set it to 10000, the previous value is {}", arrayLength);
```

Comment:
This warning message is a bit unclear. Could you change it to something like `logger.warn("The max items allowed in property {} is too large ({} items), restricting it to 10,000 items", property, arrayLength)`?